components/centraldashboard/public/components/dashboard-view.css

```
@@ -52,25 +75,11 @@ paper-card.thin {
     grid-column: 3
 }
 
-#Quick-Links .link {
-    width: 80%;
-    margin: .5em auto;
-    border: 1px solid #eeeeef;
-    padding: .5em 1em;
-    border-radius: 5px;
-    @apply --layout-horizontal;
-}
-
-#Quick-Links .link.more-coming {
-    opacity: .4;
-    font-style: italic;
-    pointer-events: none
-}
-
-#Quick-Links .link .button {
-    color: var(--accent-color);
-    background: rgba(0, 125, 252, 0.25);
-    border-radius: 50%
+#Platform-Links header {
+    font-size: 18px;
+    font-weight: 400;
+    font-family: var(--paper-card-header_-_font-family);
+    padding: 8px 16px;
```

```suggestion
    padding: .5em 1em;
```

As suggested above to replace `px` with `em`

---

Done.